### PR TITLE
ign -> gz CMake, Python, Partial Source, and File Migrations : sdformat

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/765

### CODE REVIEW

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

File: python/test/pyModel_TEST.py [hunk 1]
```diff
@@ -251,7 +251,7 @@ def test_add_modify_frame(self):
 
     def test_uri(self):
         model = Model()
-        uri = "https:#fuel.ignitionrobotics.org/1.0/openrobotics/models/my-model"
+        uri = "https:#fuel.gzrobotics.org/1.0/openrobotics/models/my-model"
```
Comment (scpeters) [inline python/test/pyModel_TEST.py:254]: this should be reverted

Comment (methylDragon) [reply python/test/pyModel_TEST.py]: Oops, reverted!

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Pull request merged**